packages/react/src/runtimes/edge/createEdgeRuntimeAPI.ts

```
@@ -119,7 +120,29 @@
     tools: lmServerTools.concat(clientTools as LanguageModelV1FunctionTool[]),
     ...(toolChoice ? { toolChoice } : undefined),
   });
-  stream = streamResult.stream;
+  stream = streamResult.stream.pipeThrough(new LanguageModelV1StreamDecoder());
+
+  const [t1, t2] = stream.tee();
+  stream = t1;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)